insights-unregistered.service.in

```diff
@@ -0,0 +1,22 @@
+# This file is part of insights-client.
+#
+# Any changes made to this file will be overwritten during a software update. To
+# override a parameter in this file, create a drop-in file, typically located at
+# /etc/systemd/system/insights-unregister.service.d/override.conf. Put the
```

Comment:
If we're going to have a comment to this effect, we should make sure the recommended path is correct. For this unit, to override directives, the path needs to be `/etc/systemd/system/insights-unregistered.service.d/override.conf`.

Reply:
bad copy&paste